include/umf/memory_pool_ops.h

```diff
@@ -131,16 +131,24 @@ typedef struct umf_memory_pool_ops_t {
     ///        The function is used to perform various control operations
     ///        on the memory pool.
     ///
-    /// @param hPool handle to the memory pool.
+    /// @param pool handle to the memory pool.
     /// @param operationType type of the operation to be performed.
     /// @param name name associated with the operation.
     /// @param arg argument for the operation.
+    /// @param size size of the argument [optional - check path requirements]
     /// @param queryType type of the query to be performed.
     ///
     /// @return umf_result_t result of the control operation.
     ///
-    umf_result_t (*ctl)(void *hPool, int operationType, const char *name,
-                        void *arg, umf_ctl_query_type_t queryType);
+    umf_result_t (*ctl)(void *pool, int operationType, const char *name,
+                        void *arg, size_t size, umf_ctl_query_type_t queryType);
+
+    ///
+    /// @brief Get the name of the memory pool.
+    /// @param pool pointer to the memory pool
+    /// @return name of the memory pool
+    ///
+    const char *(*get_name)(void *pool);
```

Comment:
Document this as optional - so this ptr can be null

ensure if pool do not support this call, umfPoolName returns unsupported error. Do it similar way to providers.

Reply:
document it how it must work with null ptr.

Reply:
Done